Classes/KIFUITestActor.h

```diff
@@ -50,6 +50,22 @@ typedef NS_ENUM(NSUInteger, KIFStepperDirection) {
 	KIFStepperDirectionDecrement
 };
 
+/*!
+ @enum KIFStepperPullToRefreshTiming
+ @discussion The approximate time in which the pull to refresh travels downward.
+ @abstract Timing in which to perform the pull down before release.
+ @constant KIFPullToRefreshInAboutAHalfSecond about half a second
+ @constant KIFPullToRefreshInAboutOneSecond about one second
+ @constant KIFPullToRefreshInAboutTwoSeconds about two seconds
+ @constant KIFPullToRefreshInAboutThreeSeconds about three seconds.
+ */
+typedef NS_ENUM(NSUInteger, KIFStepperPullToRefreshTiming) {
```

Comment:
Thanks for your keen eye!



KIFStepperPullToRefreshTiming





-----Original Message-----
From: phatmann <notifications@github.com>
To: kif-framework/KIF <KIF@noreply.github.com>
Cc: mikelupo <mikelupo@aol.com>
Sent: Fri, Sep 25, 2015 6:43 am
Subject: Re: [KIF] Add new pullToRefresh methods with associated label and identifier tests (#722)



In Classes/KIFUITestActor.h:
> @@ -50,6 +50,22 @@ typedef NS_ENUM(NSUInteger, KIFStepperDirection) {
>  	KIFStepperDirectionDecrement
>  };
>  
> +/*!
> + @enum KIFStepperPullToRefreshTiming
> + @discussion The approximate time in which the pull to refresh travels downward.
> + @abstract Timing in which to perform the pull down before release.
> + @constant KIFPullToRefreshInAboutAHalfSecond about half a second
> + @constant KIFPullToRefreshInAboutOneSecond about one second
> + @constant KIFPullToRefreshInAboutTwoSeconds about two seconds
> + @constant KIFPullToRefreshInAboutThreeSeconds about three seconds.
> + */
> +typedef NS_ENUM(NSUInteger, KIFStepperPullToRefreshTiming) {

I doubt you meant to call this KIFStepperPullToRefreshTiming. Looks like a copy/paste error from the enum above :-)
—
Reply to this email directly or view it on GitHub.